fern/pages/servers/express.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Express Server
+description: Generate Express Node.js server code from an API specification including OpenAPI, Swagger, AsyncAPI, and Fern Definition.
+subtite: Generate types and networking logic for your Node.js Express server.
+slug: express
+---
+
+Express is a minimal and flexible Node.js web application framework. It is a popular choice for building APIs and web servers.
+
+Fern generates TypeScript types and networking logic for your Express server. This saves you time and adds compile-time safety by guaranteeing that you are serving the exact API that you specified in your API definition.
+
+## What Fern generates
+
+- TypeScript interfaces for your API types
+- Exceptions that you can throw for non-200 responses
+- Interfaces for you to define your business logic
+- All the networking/HTTP logic to call your API
+
+## Use the starter repository 
+
+Clone the [starter repo](https://github.com/fern-api/express-starter/) for Express + React. 
+
+## Demo video
+
+Learn how Fern can be helpful when building an Express server. 
+
+<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/31f4243c4d824c54938bdc4840fbb8ba?sid=3f460e48-af22-4db3-be2f-48884ab0170d" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Getting started 
+
+<Steps>
+
+### Define your API 
+
+Define your API using the [OpenAPI Specification](/learn/api-definition/openapi/overview) or the [Fern Definition](/learn/api-definition/fern/overview).
+
+### Add the Express generator 
+
+Add the Express server generator to your `generators.yml` file.
+
+<CodeBlock title="generators.yml">
+```yaml
+groups:
+  express-server:
+    generators:
+      - name: fernapi/fern-typescript-express
+        version: 0.15.0
+        output:
+            location: local-file-system
+            path: ../app/server
+```
+</CodeBlock>
+
+### 
+Run the [CLI](/learn/cli-api/cli-reference/cli-overview) command `fern generate --group express-server`. 
+
+<Note>Make sure to enable `allowSyntheticDefaultImports` in your `tsconfig.json` when using this generator.</Note>
+
+### Implement the functions 
+
+Provide the business logic by implementing each function. For example, if we were implenting the IMDb API, we would implement the `get_movie` function.
+
+<CodeBlock title="src/movies_service.py">
+```python
+from .generated.fern import AbstractMoviesService, Movie, MovieDoesNotExistError, MovieId
```

Comment:
While I’m here 🫣 this file should prob have express examples not fastapi examples